apps/vr-tests-react-components/src/utilities/TestWrapperDecorator.tsx

```diff
@@ -61,23 +53,6 @@ const useNoAnimationStyles = makeStyles({
     },
   },
 });
-export const TestWrapperDecoratorNoAnimation: DecoratorFunction<ExtendedStoryFnReturnType> = story => {
-  const noAnimationStyles = useNoAnimationStyles();
-  const className = mergeClasses(noAnimationStyles.root, 'testWrapper');
-  return (
-    <div style={{ display: 'flex' }}>
```

Comment:
It looks like the `TestWrapperDecoratorNoAnimation` used flex layout and `TestWrapperDecoratorPauseAnimation` used grid layout, which is causing a bunch of extra changes in the screenshots. 

Given that the standard `TestWrapperDecorator` uses flex layout, perhaps `TestWrapperDecoratorPauseAnimation` should be updated to match it?

Reply:
Probably best to be consistent and use flex.

> which is causing a bunch of extra changes in the screenshots.

I wonder if this is a good reason to test with different layout techniques? I don't know that we can control for every possible layout but it seems that Fluent controls should render consistently in flex and grid layouts. WDYT?

Reply:
The vr-test does not show up when `didplsy: flex`

Reply:
It looks like TestWrapperDecoratorPauseAnimation is still using grid, below?

Reply:
I assume the grid layout is the reason that all of the screenshots are very large, and the spinner stories are full width. Hopefully using flex will make them smaller.